torch/csrc/distributed/rpc/init.cpp

```diff
@@ -263,9 +263,11 @@ PyObject* rpc_init(PyObject* _unused, PyObject* noargs) {
               )")
           .def(
               "rpc_sync",
-              [](const PyRRef& self) {
-                return self.createRRefProxy(RRefProxyType::RPC_SYNC);
+              [](const PyRRef& self, float timeoutSeconds) {
```

Comment:
shouldn't this be `int64_t` or or std::chrono::milliseconds instead of float ?

Reply:
I think this is float so that we can support it in torchscript? I think we used float for regular rpc timeout as well for this reason?

Reply:
Yes, I believe the convention we're using in RPC code is for timeout to be a float indicating the number of seconds. There was a PR to make that consistent across the codebase a while back. The float allows for a fractional timeout.

Reply:
hmmm i see, yeah if rpc timeouts are already float, then let's follow that convention.